src/components/App/App.js

```diff
@@ -29,22 +30,25 @@ export default class App extends React.Component {
       onSearchChange,
       searchValue,
       onLogoClick,
-      resultsValue
+      resultsValue,
+      loadingResults
     };
   }
 
   onSearchChange = value => {
+    this.setState({loadingResults: true});
     getRepos(value)
       .then(res => {
-        this.setState({ searchValue : value, resultsValue : res });
-      }).catch(::console.error);
+        this.setState({ searchValue : value, resultsValue : res , loadingResults: false});
+      })
```

Comment:
looks like you're not handling the rejection case here, and thus loading spinner will spin indefinitely when something goes wrong (e.g. github is rate-limiting you?)